Content.Server/CartridgeLoader/Cartridges/CrimnialRecordsCartridgeSystem.cs

```diff
@@ -0,0 +1,65 @@
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Shared.CriminalRecords;
+using Content.Shared.StationRecords;
+using Content.Shared.Security;
+using Content.Server.StationRecords.Systems;
+using Content.Server.Station.Systems;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class CriminalRecordsCartridgeSystem : EntitySystem
+{
+    [Dependency] private readonly CartridgeLoaderSystem? _cartridgeLoaderSystem = default!;
+    [Dependency] private readonly StationRecordsSystem _stationRecords = default!;
+    [Dependency] private readonly StationSystem _stationSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<CriminalRecordsCartridgeComponent, CartridgeUiReadyEvent>(OnUiReady);
+    }
+
+    private void OnUiReady(
+      EntityUid uid,
+      CriminalRecordsCartridgeComponent criminalRecordsCartridge,
```

Comment:
In my last PR, @TheShuEd mentioned that I should use the ``Entity<T>`` syntax here. 
This is my first time working with systems like so I'm not entirely sure about why I would do that, or what I would have to change in the code. 

Any suggestions?

Reply:
It's a pretty easy change.

```C#
private void OnUiReady(Entity<CriminalRecordsCartridgeComponent> entity, ref CartridgeUiReadyEvent args)
```
`entity` will cast to EntityUid, so just use that wherever you had uid. You can get the component with Entity.Comp.